pkgs/applications/terminal-emulators/byrd/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, vte, libconfig, glib, pkg-config }:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, vte, libconfig, glib, pkg-config }:
```

Reply:
This is new. I've always used stdenv.lib; when was this change introduced?

Reply:
Yeah, it is new. The change was 18 days ago https://github.com/NixOS/nixpkgs/commit/c06b2b3d671da4847f950e7e9041920b40eca0bf#diff-b0739f44f25865ddf604159bdc99cc82f3c192f46b92215da2594d3e5747f0b0.